src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -175,31 +175,31 @@ public function normalize($data, $format = null, array $context = [])
      *
      * @throws NotNormalizableValueException
      */
-    public function denormalize($data, $type, $format = null, array $context = [])
+    public function denormalize($data, string $class, string $format = null, array $context = [])
```

Comment:
renaming for a lower branch if confirmed

Reply:
I've renamed the parameter in order to match the interface.